models/breast_density_classification/configs/metadata.json

```diff
@@ -1,10 +1,10 @@
 {
     "version": "0.1.0",
-    "monai_version": "1.0.0rc1",
+    "monai_version": "1.0.0",
     "pytorch_version": "1.10.0",
     "numpy_version": "1.21.2",
     "optional_packages_version": {
-        "pydicom": "3.2.1"
+        "torchvision": "0.11.0"
```

Comment:
It seems pydicom is not used, thus I removed it. Fix me if it may be useful. I also added torchvision and changed the monai version into a formal release version.

Reply:
I kept the pydicom in order to ensure that if someone uses a dicom image.